components/crud-web-apps/jupyter/Dockerfile

```diff
@@ -43,9 +43,8 @@ RUN pip3 install -r requirements.txt
 
 COPY ./jupyter/backend/apps/ ./apps
 COPY ./jupyter/backend/entrypoint.py .
-COPY ./jupyter/backend/Makefile .
 
 COPY --from=frontend /src/dist/default/ /src/apps/default/static/
 COPY --from=frontend /src/dist/rok/ /src/apps/rok/static/
 
-ENTRYPOINT make run
+ENTRYPOINT ["/bin/bash","-c","APP_PREFIX=/jupyter gunicorn -w 3 --bind 0.0.0.0:5000 --access-logfile - entrypoint:app"]
```

Comment:
Please remove the `/bin/bash -c` prefix and don't set the `APP_PREFIX` ENV var

Reply:
@kimwnasptd Do I need to test if this works, or have you already used it with this change?

Reply:
@kimwnasptd removing the "/bin/bash" seems to break things as well as it doesn't know what to execute.

Reply:
I'm not sure I understand what you mean.
What is the entrypoint you tried?

Reply:
I tried what you suggested (on a different code base, but derived from this app) and got command not found in container startup. 

What I tried:
ENTRYPOINT ["gunicorn -w 3 --bind 0.0.0.0:5000 --access-logfile - entrypoint:app"]

Reply:
Ah I see what's going on. I had in mind the _shell_ form of the entrypoint
```
ENTRYPOINT gunicorn -w 3 --bind 0.0.0.0:5000 --access-logfile - entrypoint:app
```
which is the exact same thing as using the _exec_ form starting with `["/bin/bash", "-c", ...]`

It's OK to leave the list as you have it to make it explicit we use the shell form. Could you just remove the ENV var? I prefer if we set it in the manifests instead of providing a value for it inside the image

Reply:
Makes sense. I pushed the change with the APP_PREFIX removed.